src/components/AddTodoForm.js

```diff
@@ -1,6 +1,10 @@
 import React from "react";
 import InputWithLabel from "./InputWithLabel";
+import PropTypes from "prop-types";
 
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func.isRequired,
+};
 function AddTodoForm(props) {
```

Comment:
You should have a space between proptypes and the function, but you can make that change in your final project.